### PR TITLE
ft: ZENKO-476 Add build image step to pre-merge for eve

### PR DESCRIPTION
**Pull request by anurag4DSB**
Changes:

- Add a docker build step in premerge to check if the images are build properly
- Created a new step called post-merge. This step will only be triggered when a pull request is merged into development/* branches.

EVE test passed: https://eve.devsca.com/github/scality/backbeat/#/builders/5/builds/2656
TODO: Once we get the credentials from the Bert-e team we need to update the post merge step.

### CODE REVIEW

**Review by tmacro — DISMISSED** [review]

**Comment by rahulreddy** [comment]
@bert-e wait

File: eve/main.yml [hunk 1]
```diff
@@ -2,18 +2,31 @@
 version: 0.2
```
Comment (rahulreddy) [inline eve/main.yml:2]: Can we use the latest version of eve?

Comment (anurag4DSB) [reply eve/main.yml:2]: @rahulreddy checked with @tcarmet v0.2 is the latest

File: eve/main.yml [hunk 2]
```diff
@@ -2,18 +2,31 @@
...
-    worker:
+    worker: &pod
+      type: kube_pod
+      path: eve/workers/pod.yml
+    steps:
+    - TriggerStages:
+        name: trigger all the tests
+        stage_names:
+        - run-tests
+        - docker-build
```
Comment (ssalaues) [inline eve/main.yml:21]: I thought we we only doing the docker-build post-merge and not during pre-merge ??

Comment (ssalaues) [reply eve/main.yml:21]: Nevermind! Just saw the note about testing to see if it will build properly =)

Comment (anurag4DSB) [reply eve/main.yml:21]: @ssalaues Thank you

**Comment by anurag4DSB** [comment]
rebased branch

**Review by tmacro — APPROVED** [review]

**Review by philipyoo — APPROVED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Pull request merged**